spec/functional/outbox.spec.js

```diff
@@ -465,37 +473,37 @@ describe('outbox', function () {
         expect(activities[0].object[0]).toEqual(expectedObj)
         expect(activities[1].object[0]).toEqual(expectedObj)
       })
-      it('adds updated object recipients to audience')
-      it('federates whole updated object', async function (done) {
+      // it('adds updated object recipients to audience')
```

Comment:
The empty `it` leaves a "pending" test in the results as a TODO reminder
```suggestion
      it('adds updated object recipients to audience')
```